Shifty.App/Shared/NavMenu.razor

```diff
@@ -7,6 +7,8 @@
         <MudNavLink Href="MenuItems" Match="NavLinkMatch.Prefix" Icon="@Icons.Material.Outlined.Coffee">Menu Item Management</MudNavLink>
         <MudNavLink Href="Users" Match="NavLinkMatch.Prefix" Icon="@Icons.Material.Filled.ManageAccounts">Manage users</MudNavLink>
     }
+    <MudDivider />
+    <MudNavLink ForceLoad="true" Href="Logout" Match="NavLinkMatch.Prefix" Icon="@Icons.Material.Filled.Logout">Logout</MudNavLink>
```

Comment:
Does it cause extra navigation/ refreshes for the user that we are sending them to a page that does the logout process? 

Was thinking if it made sense to call the method directly from here, if it affects the user experience currently. If it doesn't, then let's keep it as is.

Reply:
You're right.

I think this implementation came to be from incrementally trying things to get it to work and this was the first version that worked without weird UI behaviour or having to reload manually an extra time.

Only downside from just having this as an `OnClick="_authenticationService.Logout"`, is that it is converted into a `div` element with no `href` attribute, so I am assuming this will have an effect on screen-readers / browser extensions. (Example: My vimium does not recognize it as a button, so I cannot use my keyboard to click it or tab to it. 
The CSS still uses the `pointer` cursor similar to the navigation links others.